jjb/ci-management/git-semver.yaml

```diff
@@ -0,0 +1,12 @@
+# git-semver.xml
```

Comment:
Wrong file extension. I don't actually see the need for this line at all.

Reply:
dunno where i picked that up from